src/content/quickstart/kind/_index.md

```diff
@@ -60,12 +60,25 @@ To manually verify the deployment follow the steps below.
 ```bash
 kubectl --kubeconfig output/kubeconfigs/kind-config-cluster3 create deployment nginx --image=nginx
 kubectl --kubeconfig output/kubeconfigs/kind-config-cluster3 expose deployment nginx --port=80
-kubectl --kubeconfig output/kubeconfigs/kind-config-cluster2 run --generator=run-pod/v1 tmp-shell --rm -i --tty --image quay.io/submariner/nettest -- /bin/bash
-curl nginx
+kubectl --kubeconfig output/kubeconfigs/kind-config-cluster3 -n default apply -f - <<EOF
+apiVersion: lighthouse.submariner.io/v2alpha1
+kind: ServiceExport
+metadata:
+  name: nginx
+EOF
+kubectl --kubeconfig output/kubeconfigs/kind-config-cluster2 -n default  run --generator=run-pod/v1 tmp-shell --rm -i --tty --image quay.io/submariner/nettest -- /bin/bash curl nginx.default.svc.supercluster.local:8080
 ```
 
-Alternately, you can also run the Submariner test-suite that validates various use-cases: `subctl verify-connectivity`
+{{% notice info %}}
 
+The `ServiceExport` resource must be created in the same namespace as the Service you're trying to export. In the example above, `nginx` is created in `default` so we create the `ServiceExport` resource in `default` as well.
+
+{{% /notice %}}
+
+Alternately, you can also perform automated verification by running the Submariner test-suite that validates various use-cases: `subctl verify`
```

Comment:
```suggestion
Alternately, you can also perform automated verifications of service discovery via the `subctl verify` command.
```
Since the next section covers this, I suggest to move this sentence and replece L81.